### PR TITLE
Improve accessibility

### PR DESCRIPTION
**Pull request by KirilCycle**
# Please use the matching pull request template

Choose a template by going into `Preview` and then clicking on the link.
It will reload this page and fill it with a checklist to help you make
the pull request as good as possible.

## Available Templates

- [Framework][Framework] changes to the framework
- [Tools][Tools] changes to anything in the tools directory
- [Translations][Translations] changes to the translation files
- [Documentation-Website][DW] changes to the website
- [History-Website][HW] changes to the history-ui
- [History-Microservice][HM] changes to the history-microservice
- [Documentation][Documentation] changes to a md file (Readme for example)
- [Examples][Examples] adding or adjusting examples
- [Dockerfiles][DF] adding or modifying dockerfiles
- [Mindmap][MM] adding to or modifying the mindmap
- [CLI][CLI] adding to or modifying the cli
- [Full][Full] changes to multiple categories

[Full]: ?quick_pull=1&template=full.md
[Documentation]: ?quick_pull=1&template=documentation.md
[HM]: ?quick_pull=1&template=history-microservice.md
[HW]: ?quick_pull=1&template=history-website.md
[DW]: ?quick_pull=1&template=documentation-website.md
[Translations]: ?quick_pull=1&template=translations.md
[Tools]: ?quick_pull=1&template=tools.md
[Framework]: ?quick_pull=1&template=framework.md
[Examples]: ?quick_pull=1&template=examples.md
[DF]: ?quick_pull=1&template=dockerfiles.md
[MM]: ?quick_pull=1&template=mindmap.md
[CLI]: ?quick_pull=1&template=cli.md

### CODE REVIEW

**Comment by KirilCycle** [comment]
e header component has problems, the tab clicking on the link that contains the sub links does not open, I tried to fix it, but this solution will be a bit complicated, it is better to create a new issue

File: documentation-website/src/pages/faq/index.tsx [hunk 1]
```diff
@@ -6,26 +6,27 @@ import Layout from '../../components/layout.tsx';
 import FaqItem from '../../components/faq-item.tsx';
 
 const Index = () => <Layout
+  Outlet={
```
Comment (Idrinth) [inline documentation-website/src/pages/faq/index.tsx:9]: not going to work anymore

File: documentation-website/src/components/header.tsx [hunk 2]
```diff
@@ -54,19 +54,24 @@ const Header = ({
...
         <NavLink to="/quick-start/">
           <Lang lnkey='nav.quick-start'/>
         </NavLink>
       </li>
       <li>
-        <NavLink className="dropdown-links-heading" to="/contributing/">
+        <NavLink
+          aria-expanded="false"
+          id="dropdown-contributors"
+          aria-controls="contributors-menu"
```
Comment (Idrinth) [inline documentation-website/src/components/header.tsx:68]: this is technically the contributing menu - contributors are just one item in it

**Review by Idrinth — CHANGES_REQUESTED** [review]

**Comment by Idrinth** [comment]
> # Please use the matching pull request template
> 
> Choose a template by going into `Preview` and then clicking on the link. It will reload this page and fill it with a checklist to help you make the pull request as good as possible.

would be nice if you would use the template  btw  ;)

**Review by Idrinth — APPROVED** [review]

**Pull request merged**